modules/govuk_cdnlogs/templates/etc/logrotate.cdn_logs_hourly.conf.erb

```diff
@@ -4,7 +4,7 @@
 -%>
 
 {
-  rotate 720
+  maxage 30
```

Comment:
This is an hourly log rotation, so this would rotate after 30 hours worth of rotations. I appreciate it didn't work as expected, but I think this change is incorrect.